nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/datasetbrowser/DatasetBrowserTableView.java

```diff
@@ -50,8 +52,10 @@ public DatasetBrowserTableView(boolean addCacheColumn) {
     }
 
     public void setDatasetSummaries(List<DatasetSummary> summaries) {
-        unfilteredDatasetSummaries.clear();
-        unfilteredDatasetSummaries.addAll(summaries);
+        unfilteredDatasetSummaries.setAll(summaries);
+        // This is required as a fix for header columns misalignment when tableview has a scrollbar, is in a tab
+        // and uses TableView.CONSTRAINED_RESIZE_POLICY
+        Fx.runOnFxThread(() -> scrollTo(0));
```

Comment:
There is a [bug](https://bugs.openjdk.org/browse/JDK-8090112) with the CONSTRAINED_RESIZE_POLICY (I'm using this policy to get rid of the extra empty column at the end of the table). It is supposed to be fixed with javafx20 which I haven't had a chance to see if it actually would fix this particular problem. There are several suggested workarounds but this is the only one that I could get to work.